sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/CSharpSyntaxHelper.cs

```diff
@@ -199,7 +199,7 @@ public static bool IsStringEmpty(this ExpressionSyntax expression, SemanticModel
                 ?? methodDeclarationBase?.ExpressionBody()?.DescendantNodes()
                 ?? Enumerable.Empty<SyntaxNode>();
 
-            // See issue: https://github.com/SonarSource/sonar-csharp/issues/416
```

Comment:
could you please separate the commit of changing the github URLs (for which the redirect works fine) from this PR which only changes the Maven pom files ?